website/templates/base.html

```diff
@@ -1,92 +1,144 @@
-        <link rel="shortcut icon" href="{% static 'img/favicon.ico' %}" />
-        <link rel="stylesheet" href="{% static 'css/site.min.css' %}" media="screen">
-        <script src="{% static 'js/lib/zepto.min.js' %}"></script>
-        {% if sentry_public_dsn %}
-        <script src="https://cdn.ravenjs.com/3.7.0/raven.min.js"></script>
-        <script>Raven.config('{% sentry_public_dsn %}').install()</script>{% endif %}
-{% block extrahead %}{% endblock %}
-{% if GOOGLE_ANALYTICS_KEY %}
-        <script>
-        window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
-        ga('create', '{{GOOGLE_ANALYTICS_KEY}}', 'auto');
-        ga('send', 'pageview');
-        </script>
-        <script async src='https://www.google-analytics.com/analytics.js'></script>
-{% else %}
+
+<head>
+    <meta charset="utf-8">
+    <title>{% block pagetitle %}Reconnecting Asia{% endblock %}</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="summary" content="Reconnecting Asia maps new linkages—roads, railways, and other infrastructure—that are reshaping economic and geopolitical realities across the continent. Through data curation and objective analysis, the project aims to fill Asia's infrastructure-information gap, squaring lofty ambitions with facts on the ground.">
+    <link rel="icon" type="image/png" href="{% static 'img/favicon.png' %}" />
+    <link rel="shortcut icon" href="{% static 'img/favicon.ico' %}" />
+    <link rel="stylesheet" href="{% static 'css/site.min.css' %}" media="screen">
+    <script src="{% static 'js/lib/zepto.min.js' %}"></script>
+
+    <script src="{% static 'js/lib/navigation.js' %}"></script>
+
+    {% if sentry_public_dsn %}
+    <script src="https://cdn.ravenjs.com/3.7.0/raven.min.js"></script>
+    <script>
+    Raven.config('{% sentry_public_dsn %}').install()
+    </script>{% endif %} {% block extrahead %}{% endblock %} {% if GOOGLE_ANALYTICS_KEY %}
+    <script>
+    window.ga = window.ga || function() {
+        (ga.q = ga.q || []).push(arguments)
+    };
+    ga.l = +new Date;
+    ga('create', '{{GOOGLE_ANALYTICS_KEY}}', 'auto');
+    ga('send', 'pageview');
+    </script>
+    <script async src='https://www.google-analytics.com/analytics.js'></script>
+    {% else %}
     <!-- Analytics disabled -->
-{% endif %}
-    </head>
-    <body class="{% block bodyclass %}{% endblock %}">
-        <header role="banner">
-            <div class="wrap">
-                <div class="headers">
-                    <a href="/"><h1>Reconnecting Asia</h1></a>
-                    <h2>Mapping continental ambitions</h2>
+    {% endif %}
+</head>
+
+<body class="{% block bodyclass %}{% endblock %}">
+    <div id="site-wrapper">
+        <div class="header-container-wrapper">
+            <div class="header-top-wrapper">
+                <!-- <div class="navigation">
+        <div class="headers">
```

Comment:
We'll have this in the git history so it would be my vote to remove this commented block.